Docs/source/install/hpc/adastra.rst

```diff
@@ -85,16 +85,24 @@ Finally, since Adastra does not yet provide software modules for some of our dep
 Compilation
 -----------
 
-Use the following :ref:`cmake commands <building-cmake>` to compile:
+Use the following :ref:`cmake commands <building-cmake>` to compile the application:
```

Comment:
I think we refer to it as "the application **executable**" in the rest of the text below.
```suggestion
Use the following :ref:`cmake commands <building-cmake>` to compile the application executable:
```